build.sbt

```diff
@@ -10,11 +10,16 @@ lazy val examples = project in file("examples") dependsOn (root % "test->test;co
   )
 )
 
-scalacOptions in ThisBuild := Seq("-feature", "-unchecked", "-deprecation", "-encoding", "utf8", "-Xfatal-warnings")
+scalacOptions in ThisBuild := Seq("-feature", "-unchecked", "-deprecation", "-encoding", "utf8")
```

Comment:
Fatal warnings is infeasible in 2.13 since there are a lot of, unsolvable while supporting 2.10-2.12, deprecation warnings.